src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-number;
+  flex-wrap: wrap;
+  display: flex;
+  width: $my-var;
+  gap: $width-gap;
+
+  .calendar__day {
+    background-color: #eee;
+    border: 1px solid #000;
+    width: 100px;
+    height: 100px;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition:
+      transform 0.5s,
+      background-color 0.5s;
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+    }
+  }
+  
+  .calendar__day::before {
+    content: counter(day-number);
+    counter-increment: day-number;
+    color: #000;
+  }
+
+  @each $day, $n in (mon: 0, tue: 1, wed: 2, thu: 3, fri: 4, sat: 5, sun: 6) {
+    &--days-#{$day} &__day:first-child {
+      margin-left: ($width-boxs + $width-gap) * $n;
```

Comment:
The variables `$width-boxs` and `$width-gap` are not descriptive enough. Consider renaming them to something more meaningful, like `$calendar-day-size` and `$calendar-day-gap`.